src/patternfly/components/DualListSelector/dual-list-selector-draggable-button.hbs

```diff
@@ -1,9 +1,9 @@
 {{#if dual-list-selector-draggable-button--IsDisabled}}
-  {{#> span button--modifier="pf-m-plain" button--attribute=(concat 'disabled aria-pressed="false" aria-label="Reorder" id="' dual-list-selector-list-item--id '-draggable-button" aria-labelledby="' dual-list-selector-list-item--id '-draggable-button ' dual-list-selector-list-item--id '-item-text" aria-describedby="' dual-list-selector--id '-help"')}}
+  {{#> button button--type="span" button--modifier="pf-m-plain" button--attribute=(concat 'disabled aria-pressed="false" aria-label="Reorder" id="' dual-list-selector-list-item--id '-draggable-button" aria-labelledby="' dual-list-selector-list-item--id '-draggable-button ' dual-list-selector-list-item--id '-item-text" aria-describedby="' dual-list-selector--id '-help"')}}
     <i class="fas fa-grip-vertical" aria-hidden="true"></i>
-  {{/span}}
+  {{/button}}
 {{else}}
-  {{#> span button--modifier="pf-m-plain" button--attribute=(concat 'aria-pressed="false" aria-label="Reorder" id="' dual-list-selector-list-item--id '-draggable-button" aria-labelledby="' dual-list-selector-list-item--id '-draggable-button ' dual-list-selector-list-item--id '-item-text" aria-describedby="' dual-list-selector--id '-help"')}}
+  {{#> button button--type="span" button--modifier="pf-m-plain" button--attribute=(concat 'aria-pressed="false" aria-label="Reorder" id="' dual-list-selector-list-item--id '-draggable-button" aria-labelledby="' dual-list-selector-list-item--id '-draggable-button ' dual-list-selector-list-item--id '-item-text" aria-describedby="' dual-list-selector--id '-help"')}}
```

Comment:
![Screenshot 2023-02-13 at 3 16 25 PM](https://user-images.githubusercontent.com/26305082/218565453-0a258453-3aab-4f46-8439-a99535dc7362.png)
In react, i don't see an aria-describedby, aria-label, aria-labelledby. it only uses aria-disabled="false" and the two classes (pf-c-button and pf-m-plain). It's functionally not a button, just a visual indicator that the row is draggable.
I think removing these extra labels will resolve the a11y violation?
FWIW, it looks like the [draggable dual list selector example](https://www.patternfly.org/v4/components/dual-list-selector/html) is broken in production, which is fine since I'm making updates to the dragdrop anyway and we might move the examples out of table into a dragdrop demo instead or something similar. I don't think dragdrop examples need to be perfect today.